build/config/AssemblySignToolData.json

```diff
@@ -4,45 +4,35 @@
       "certificate": "Microsoft",
       "strongName": "StrongName",
       "values": [
-        "net40\\bin\\FSharp.Core.dll",
-        "net40\\bin\\*\\FSharp.Core.resources.dll",
-        "net40\\bin\\FSharp.Build.dll",
-        "net40\\bin\\*\\FSharp.Build.resources.dll",
-        "net40\\bin\\FSharp.Compiler.Private.dll",
-        "net40\\bin\\*\\FSharp.Compiler.Private.resources.dll",
-        "net40\\bin\\FSharp.Compiler.Server.Shared.dll",
-        "net40\\bin\\FSharp.Compiler.Interactive.Settings.dll",
-        "net40\\bin\\*\\FSharp.Compiler.Interactive.Settings.resources.dll",
-        "net40\\bin\\fsc.exe",
-        "net40\\bin\\fsi.exe",
-        "net40\\bin\\fsiAnyCpu.exe",
-        "net40\\bin\\FSharp.VS.FSI.dll",
-        "net40\\bin\\*\\FSharp.VS.FSI.resources.dll",
-        "net40\\bin\\FSharp.LanguageService.Base.dll",
-        "net40\\bin\\*\\FSharp.LanguageService.Base.resources.dll",
-        "net40\\bin\\FSharp.LanguageService.dll",
-        "net40\\bin\\*\\FSharp.LanguageService.resources.dll",
-        "net40\\bin\\FSharp.UIResources.dll",
-        "net40\\bin\\*\\FSharp.UIResources.resources.dll",
-        "net40\\bin\\FSharp.Editor.dll",
-        "net40\\bin\\*\\FSharp.Editor.resources.dll",
-        "net40\\bin\\FSharp.ProjectSystem.Base.dll",
-        "net40\\bin\\*\\FSharp.ProjectSystem.Base.resources.dll",
-        "net40\\bin\\FSharp.ProjectSystem.PropertyPages.dll",
-        "net40\\bin\\*\\FSharp.ProjectSystem.PropertyPages.resources.dll",
-        "net40\\bin\\FSharp.ProjectSystem.FSharp.dll",
-        "net40\\bin\\*\\FSharp.ProjectSystem.FSharp.resources.dll",
-        "net40\\bin\\FSharp.PatternMatcher.dll",
-        "coreclr\\bin\\FSharp.Core.dll",
-        "coreclr\\bin\\*\\FSharp.Core.resources.dll",
-        "coreclr\\bin\\FSharp.Build.dll",
-        "coreclr\\bin\\*\\FSharp.Build.resources.dll",
-        "coreclr\\bin\\FSharp.Compiler.Private.dll",
-        "coreclr\\bin\\*\\FSharp.Compiler.Private.resources.dll",
-        "coreclr\\bin\\FSharp.Compiler.Interactive.Settings.dll",
-        "coreclr\\bin\\*\\FSharp.Compiler.Interactive.Settings.resources.dll",
-        "coreclr\\bin\\fsc.exe",
-        "coreclr\\bin\\fsi.exe"
+        "FSharp.Core\\*\\*\\FSharp.Core.dll",
```

Comment:
Can `/` be used instead?

Reply:
Probably.  I'm trying to keep the changes in this PR minimal so I don't have so many moving parts to track, but I've filed #6023.